client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts

```diff
@@ -101,7 +95,7 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
             investigation: epidemiologyNumber
         })
         interactedContactsLogger.info(`launching server request with epidemiology number ${epidemiologyNumber}`, Severity.LOW);
-        axios.get(`/clinicalDetails/coronaTestDate/${epidemiologyNumber}`).then((res: any) => {
+        axios.get('/clinicalDetails/coronaTestDate').then((res: any) => {
```

Comment:
why not sending epidemiologyNumber?

Reply:
Sending the epidemiologyNumber is no necessary because we get it from response.locals